components/create_comment/create_comment.jsx

```diff
@@ -341,6 +341,9 @@ export default class CreateComment extends React.PureComponent {
 
         if (!Utils.cmdOrCtrlPressed(e) && !e.altKey && !e.shiftKey && Utils.isKeyPressed(e, Constants.KeyCodes.UP) && message === '') {
             e.preventDefault();
+            if (this.refs.textbox) {
+                this.refs.textbox.blur();
+            }
```

Comment:
Similar to the fix in #850, need to manually blur so closing the edit modal does not autofocus the texbox when we want to focus the delete button on the newly opened modal